cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -95,11 +91,15 @@ def get_params_for_user_creating(_user):
         if users is None:
             users = self.config.users
         for user in users:
+            self.log.debug("Creating user with name %s.", user['name'])
             self.keystoneclient.users.create(
                 **get_params_for_user_creating(user))
             if not user.get('additional_tenants'):
                 continue
             for tenant in user['additional_tenants']:
+                msg = "Adding user {} with role {} to tenant {}.".format(
+                    user['name'], tenant['role'], tenant['name'])
```

Comment:
Please don't use "...".format(x, y, z) for logging. Use `self.log.debug("Adding user %s with role %s to tenant %s.", user['name'], tenant['role'], tenant['name'])` for formatting.